src/main/java/nextstep/courses/domain/session/SessionType.java

```diff
@@ -0,0 +1,10 @@
+package nextstep.courses.domain.session;
+
+public enum SessionType {
+  FREE,
+  PAID;
```

Comment:
```suggestion
  FREE("무료"), // 설명을 포함
  PAID("유료");
```
이 부분은 의견이 엇갈릴 수 있는데요
저는 enum에도 되도록이면 설명을 담아주려고합니다.
이유는 다음과 같아요

- 코드성이고 영문이기 때문에 타입이 많아졌을 때 바로 파악하기 어려울 수 있다.
- enum도 코드의 일부분이므로 리팩토링 대상이 될 수 있어 enum name이 변경될 수 있다.

참고만 해주세요~